build/charts/antrea/templates/agent/daemonset.yaml

```diff
@@ -99,6 +99,8 @@ spec:
             {{- end }}
             # SKIP_CNI_BINARIES takes in values as a comma separated list of
             # binaries that need to be skipped for installation, e.g. "portmap, bandwidth".
+            - name: GOCOVERDIR
```

Comment:
there is no reason to define this environment variable except when collecting coverage. So this should be conditioned by `if ((.Values.testing).coverage)` (see below)